portfolio/src/main/webapp/script.js

```diff
@@ -134,8 +134,8 @@ function createComment(comment) {
  * Takes a date in epoch milliseconds and returns it in the string
  * format MonthName DayNumber, YearNumber (e.g. 'January 1, 1970')
  */
-function formatDate(epochDate) {
-  const dateTimeFormat = new Intl.DateTimeFormat('en', 
+function formatDate(epochDate, lang) {
+  const dateTimeFormat = new Intl.DateTimeFormat(lang, 
```

Comment:
I think the latin month names are being translated incorrectly, but I don't know if it's something in this block of code here or due to the API. Maybe add a comment here to clarify what the list of objects does with relation to the DateTimeFormat? It might also be useful to the reader to translate the date and message in the same place.

Reply:
Yeah, I think it might be an issue with Intl.DateTimeFormat just not supporting Latin dates, but I'm not quite sure yet. 
I'll make those readability changes :)